tests/providers/docker/decorators/test_docker.py

```diff
@@ -60,6 +60,23 @@ def f(num_results):
         assert isinstance(result, list)
         assert len(result) == 50
 
+    def test_basic_docker_operator_with_template_fields(
+        , dag_maker):
+        @task.docker(image="python:3.9-slim", container_name='python_{{dag_run.dag_id}}', auto_remove="force")
+        def f():
+            import random
+
+            return [random.random() for _ in range(100)]
```

Comment:
Last my concern, because we actually not run this operator during this test, for avoid confusing someone int he future we might explicit show that this block should not executed during this test


```suggestion
            raise RuntimeError("Should not executed") 
```